integration-example/src/main/java/org/mache/GettingCacheClient.java

```diff
@@ -4,85 +4,83 @@
 import org.mache.examples.cassandra.CassandraExample;
 import org.mache.examples.mongo.MongoAnnotatedMessage;
 import org.mache.examples.mongo.MongoExample;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import javax.jms.JMSException;
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.concurrent.ExecutionException;
 
-/**
- * Created by jbowkett on 17/07/15.
- */
 public class GettingCacheClient {
+    private static final Logger LOG = LoggerFactory.getLogger(GettingCacheClient.class);
 
-  private enum CacheType {Cassandra, Mongo}
+    private enum CacheType {Cassandra, Mongo}
 
-  public static void main(String...commandLine) {
-    final Args args = parseArgs(commandLine);
-    final int count = args.count;
-    switch(args.cacheType){
-      case Cassandra:
-        doCassandra(count);
-        break;
-      case Mongo:
-        doMongo(count);
-        break;
+    public static void main(String... commandLine) {
+        final Args args = parseArgs(commandLine);
+        final int count = args.count;
+        switch (args.cacheType) {
+            case Cassandra:
+                doCassandra(count);
+                break;
+            case Mongo:
+                doMongo(count);
+                break;
```

Comment:
Only if you've got time, I don't consider the couch stuff done yet. Want to get a good handle on blueprint and fix the couchbase images. I think it's a straightforward change to a kubernetes.json file but am creating my own blueprint env at home so i can touch things without breaking them